docs/source/using/addons.adoc

```diff
@@ -273,5 +273,19 @@ After you define the add-on, you can install it by running:
 $ minishift addons install <ADDON_DIR_PATH>
 ----
 
+TIP: You can also write metadata in multiple line.
+[[example-multiline-metadata]]
+.Example: Add-on definition which contain multiline description
+
+----
+# Name: prometheus
+# Description: This template creates a Prometheus instance preconfigured to gather OpenShift and
+#  Kubernetes platform and node metrics and report them to admins. It is protected by an
```

Comment:
+1 to that. Might have been worth to write the test this way to document the default expected case. The test might imply that you need extra spaces and hence @gbraad comments.